package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "id-2-osm",
+  "version": "0.0.1",
+  "description": "A restful API for saving and querying osm entities external id's",
+  "main": "./src/index.ts",
+  "scripts": {
+    "test:unit": "jest --config=./tests/configurations/unit/jest.config.js",
+    "test:integration": "jest --config=./tests/configurations/integration/jest.config.js",
+    "lint": "eslint src/**",
+    "lint:fix": "eslint --fix src/**",
+    "release": "standard-version",
+    "test": "npm run test:unit && npm run test:integration",
+    "prebuild": "npm run clean",
+    "build": "tsc --project tsconfig.build.json  && npm run assets:copy",
+    "start": "npm run build && cd dist && node ./index.js",
+    "assets:copy": "copyfiles -f ./config/* ./dist/config && copyfiles -f ./openapi3.yaml ./dist/ && copyfiles ./package.json dist",
+    "clean": "rimraf dist"
+  },
+  "directories": {
+    "test": "tests"
+  },
+  "author": "MapColonies",
+  "license": "ISC",
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "config": {
+    "commitizen": {
+      "path": "./node_modules/cz-conventional-changelog"
+    }
+  },
+  "dependencies": {
+    "@map-colonies/mc-logger": "^1.2.1",
+    "@map-colonies/mc-model-types": "^1.0.1",
+    "@map-colonies/mc-probe": "^1.0.0",
+    "config": "^3.3.1",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "http-status-codes": "^2.1.2",
+    "js-yaml": "^3.14.1",
+    "openapi-validator-middleware": "^3.1.1",
+    "prettier": "^2.2.1",
+    "reflect-metadata": "^0.1.13",
+    "swagger-ui-express": "^4.1.4",
+    "tsyringe": "^4.3.0"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^9.1.1",
+    "@commitlint/config-conventional": "^10.0.0",
+    "@map-colonies/eslint-config": "^1.1.0",
+    "@types/config": "0.0.36",
+    "@types/express": "^4.17.8",
+    "@types/jest": "^26.0.19",
+    "@types/js-yaml": "^3.12.5",
+    "@types/supertest": "^2.0.10",
+    "@types/swagger-ui-express": "^4.1.2",
+    "@types/yamljs": "^0.2.31",
+    "commitlint": "^9.1.2",
+    "copyfiles": "^2.3.0",
+    "cz-conventional-changelog": "^3.3.0",
+    "eslint": "^7.8.1",
+    "husky": "^4.2.5",
+    "jest": "^26.6.3",
+    "jest-create-mock-instance": "^1.1.0",
+    "jest-html-reporters": "^2.0.3",
+    "nodemon": "^2.0.6",
+    "rimraf": "^3.0.2",
+    "standard-version": "^9.0.0",
+    "supertest": "^6.0.1",
+    "ts-jest": "^26.3.0",
+    "typescript": "^3.8.3"
```

Comment:
we should add prettier and remove nodemon for now until we find the proper configuration

Reply:
fixed